### PR TITLE
Small changes with no functional impact

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
Build and test passed.

- removed a leading space on the last line of test output ( from `console.log(a, b)`
- removed some static complierOptions entries which are built in ts.transpileModule()
- improved README.md section about compilerOptions

### CODE REVIEW

**Comment by PopGoesTheWza** [comment]
Typescript 3.4 features tests cnow complete with version 3.4.4:

```ts
# testTypeScript_34x_improvedReadonly
v--TS--v
// Improved support for read-only arrays and tuples
function f1(mt: [number, number], rt: readonly [number, number]) {
  mt[0] = 1;  // Ok
  // rt[0] = 1;  // Error, read-only element
}

// next function declaration crashes with Typescript version < 3.4.4
function f2(ma: string[], ra: readonly string[], mt: [string, string], rt: readonly [string, string]) {
  //   ma = ra;  // Error
  ma = mt;  // Ok
  //   ma = rt;  // Error
  ra = ma;  // Ok
  ra = mt;  // Ok
  ra = rt;  // Ok
  //   mt = ma;  // Error
  //   mt = ra;  // Error
  //   mt = rt;  // Error
  //   rt = ma;  // Error
  //   rt = ra;  // Error
  rt = mt;  // Ok
}

type ReadWrite<T> = { -readonly [P in keyof T] : T[P] };

type T0 = Readonly<string[]>;  // readonly string[]
type T1 = Readonly<[number, number]>;  // readonly [number, number]
type T2 = Partial<Readonly<string[]>>;  // readonly (string | undefined)[]
type T3 = Readonly<Partial<string[]>>;  // readonly (string | undefined)[]
type T4 = ReadWrite<Required<T3>>;  // string[]
–––
// Compiled using ts2gas 1.6.2 (TypeScript 3.4.4)
var exports = exports || {};
var module = module || { exports: exports };
// Improved support for read-only arrays and tuples
function f1(mt, rt) {
    mt[0] = 1; // Ok
    // rt[0] = 1;  // Error, read-only element
}
// next function declaration crashes with Typescript version < 3.4.4
function f2(ma, ra, mt, rt) {
    //   ma = ra;  // Error
    ma = mt; // Ok
    //   ma = rt;  // Error
    ra = ma; // Ok
    ra = mt; // Ok
    ra = rt; // Ok
    //   mt = ma;  // Error
    //   mt = ra;  // Error
    //   mt = rt;  // Error
    //   rt = ma;  // Error
    //   rt = ra;  // Error
    rt = mt; // Ok
}
^--GS--^
```

**Comment by grant** [comment]
Need the Travis to pass.

**Comment by PopGoesTheWza** [comment]
> Need the Travis to pass.

I am unfamiliar with Travis. Do you have any idea why it keeps using ts 3.4.3?

**Comment by PopGoesTheWza** [comment]
@grant I found out why this PR was stuck in Travis. `typescript` version was incorrect.

File: README.md [hunk 1]
```diff
@@ -58,10 +60,11 @@ Transpiler is called with this default *ts.TranspileOptions* object
 ```ts
 {
   compilerOptions: {
-    target: ts.ScriptTarget.ES3,
+    isolatedModules: true,
     noLib: true,
     noResolve: true,
-    module: ts.ModuleKind.None,
+    target: "ES3",
```
[the remark this answers is not in the document]
Comment (PopGoesTheWza) [reply README.md]: @grant I intentionally changed to literal values with the following reasoning:

- in order to resolve `ts.ScriptTarget.ES3` and `ts.ModuleKind.None` typescript definitions to be available, which is unlikely in the context of a json configuration file
- `ts.ScriptTarget.ES3` and `ts.ModuleKind.None` are enums which resolve into number values (0 and 0 respectively). Those values are never mentioned in typescript public documentation and could (technically though unlikely) change in a future version
- typescript documentation consistently refers to literal strings to set those properties

After reviewing those points, please confirm the request to revert

**Pull request merged**